pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -149,12 +151,20 @@ func (w *watcher) run(ctx context.Context, repo git.Repo, repoCfg config.PipedRe
 		commitMsg                  string
 		includedCfgs, excludedCfgs []string
 	)
+
+	// get rule to parse template
```

Comment:
```suggestion
	// Get rule to parse template.
```
as our team convention 😄